### ISSUE TITLE
Mass managing users bug

### ISSUE THREAD

**Issue by cherstoup** [seahindeniz/BrainlyTools_Extension#345]
**Describe the issue/bug:**

Sometimes after clicking on "mass manage users" & inputting profile links, the extension does not load. 
...

**To Reproduce:**
1. Go on Brainly.com
2. Open the moderation panel & click "mass-manage users." 
3. Paste in profile links or IDs. This should load up to allow us to mass delete profiles but sometimes it does not load anything up.

**Screenshots [paste below, optional]:**
https://prnt.sc/w6yuu5

**Comment by NisiDominus** [comment]
I get this error often too. From my observations, the error is entirely related to the Brainly API. User-based API queries are frequently returning 403 errors.
![image](https://user-images.githubusercontent.com/47514611/102752835-38ec9800-437b-11eb-9d54-881c957f9717.png)

**Comment by seahindeniz** [comment]
Yeah, that's the main issue we have and there is nothing much we can do about it because extension's `fetch` request may get rejected
